docker/k8s/tcs_config.toml

```diff
@@ -0,0 +1,96 @@
+# Copyright 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[KvStorage]
+StoragePath = "config/Kv_Shared_tmp"
+# the remote version is of higher priority if enabled
+remote_url = "http://lmdb:9090"
+
+# --------------------------------------------------
+# Logging -- configuration of service logging
+# --------------------------------------------------
+[Logging]
+LogLevel = "INFO"
+LogFile  = "__screen__"
+
+# --------------------------------------------------------------
+# EnclaveManager -- configuration of SGX Enclave EnclaveManager
+# --------------------------------------------------------------
+[EnclaveManager]
+sleep_interval = "10"
+
+[Listener]
+listener_port = 1947
+max_work_order_count = 10
+
+# --------------------------------------------------
+# EnclaveModule -- configuration of the SGX Enclave
+# --------------------------------------------------
+[EnclaveModule]
+# spid is a 32-digit hex string tied to the enclave implementation
+# Replace dummy SPID with value obtained after subscription to run TCS in SGX HW mode
```

Comment:
Trim lines to <= 80 characters for readability.

Reply:
done as 7f48d2d0a38818265784e988749c61d1b993add2